website/source/guides/security/acl.html.markdown

```diff
@@ -253,6 +253,36 @@ namespace "default" {
 }
 ```
 
+Namespace definitions may also include globs, that can be used to provide access to a category of namespaces, while allowing more specific privileges where required. For example, the below policy allows read access to most production namespaces, but allows write access to the `"production-api"` namespace, and rejects any access to the `"production-ppi"` namespace.
+
+```
+namespace "production-*" {
+    policy = "read"
+}
+
+namespace "production-api" {
+    policy = "write"
+}
+
+namespace "production-ppi" {
```

Comment:
Lets use web (or something similar); not sure what ppi is and may confuse people